src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt

```diff
@@ -63,7 +64,7 @@ class ModuleExecutor(
             } while (missingResultsPage)
             return ExecutionResults(true, failedTickets)
         } catch (ex: Throwable) {
-            log.error("Failed to execute modules", ex)
+            if (doLog) log.error("Failed to execute modules", ex)
```

Comment:
Wouldn't this prevent logging in debug mode too? IIRC you can set conditions directly in logback.xml that affect only the discord appender